inference-engine/thirdparty/clDNN/kernel_selector/core/cl_kernels/range_ref.cl

```diff
@@ -0,0 +1,11 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+KERNEL (range_ref)(const __global INPUT0_TYPE *startP, const __global INPUT1_TYPE *stopP,
+                   const __global INPUT2_TYPE *stepP, __global OUTPUT_TYPE *output)
+
+{
+    const uint i = get_global_id(2);
+    output[i] = *startP + i * (OUTPUT_TYPE) * stepP;
```

Comment:
why is it multiplying OUTPUT_TYPE?

Reply:
Was it casting ? than how about using TO_OUTPUT_TYPE instead of (OUTPUT_TYPE)?

Reply:
`OUTPUT_TYPE` is a typedef, i.e. `i` is multiplied by `*stepP` cast to `OUTPUT_TYPE`

Reply:
I mean, "TO_OUTPUT_TYPE" is macro for that casting.

Reply:
sorry, i didn't see your second comment until page reload. converted everything with `TO_OUTPUT_TYPE`